docs/pages/menus/index.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+
+      <p>Menus allow users to make a selection from a list of options. We use two types of menus:</p>
```

Comment:
RE: "We use two types of menus:" 

This colon type of text followed by big headings struck me as an incomplete sentence. Perhaps this should say:

"We use two types of menus: *Dropdown* and *Filter*."